vmodules/classes/command.ts

```diff
@@ -1,31 +1,31 @@
-import djs from 'discord.js';
-const log = require(`../util/logger.js`).write;
+import djs, { Message, PermissionString } from 'discord.js';
+import BotConfig from '../util/interfaces/BotConfig';
 
 interface ICommand {
```

Comment:
which of these properties is optional? (`property?: type`)

Reply:
description, args, subcmds, guilds, image, perm, dm, and typer are all optional.

basically everything except `name` and `run`

also since im here:
args isnt a string (i havent gotten around to doing the code for this yet, but it's gonna be an array of objects)

subcmds is gonna be an array of commands

Reply:
for now i'm gonna keep it as an array of strings until you create the feature you're talking about